ontology/yaml/resources/SAFETY/entity_types/FSS.yaml

```diff
@@ -23,3 +23,26 @@ FSS_STD:
   - FSS
   uses:
   - fire_alarm
+
+FSS_PAS:
```

Comment:
This looks like the existing type `FACP_PAS`; to follow precedent, you can add the `panel_alarm` as optional to this existing type.

Reply:
Hi ma'am, noted, will do.